wgpu-hal/src/dx12/command.rs

```diff
@@ -610,7 +626,9 @@ impl crate::CommandEncoder<super::Api> for super::CommandEncoder {
             if !ds.depth_ops.contains(crate::AttachmentOps::LOAD) {
                 flags |= native::ClearFlags::DEPTH;
             }
-            if !ds.stencil_ops.contains(crate::AttachmentOps::LOAD) {
+            if !ds.stencil_ops.contains(crate::AttachmentOps::LOAD)
+                && ds.target.view.has_stencil_aspect
```

Comment:
similarly, if we are doing this check for stencil, we need to do a similar one for depth